src/app/contexts/RequestContext/getOriginContext/index.test.js

```diff
@@ -14,28 +14,21 @@ describe('getOriginContext', () => {
     {
       bbcOrigin: undefined,
       location: undefined,
-      expected: { isUK: true, origin: 'https://www.bbc.co.uk' },
+      expected: 'https://www.bbc.co.uk',
       assertion: 'should return defaults if no origin can be found',
     },
     {
       bbcOrigin: 'https://foobar.com',
       location: 'https://beepboop.co.uk',
-      expected: { isUK: false, origin: 'https://foobar.com' },
-      assertion:
-        'should return isUK and origin based off of bbcOrigin if provided',
+      expected: 'https://foobar.com',
+      assertion: 'should return origin based off of bbcOrigin if provided',
     },
     {
       bbcOrigin: undefined,
       location: 'https://beepboop.com',
-      expected: { isUK: false, origin: 'https://beepboop.com' },
+      expected: 'https://beepboop.com',
       assertion:
-        'should return isUK and origin based off of location if bbcOrigin isnt provided',
-    },
-    {
-      bbcOrigin: 'https://beepboop.org',
-      location: 'https://beepboop.org',
-      expected: { isUK: true, origin: 'https://beepboop.org' },
-      assertion: 'should return isUK as true if tld isnt .com',
```

Comment:
Removed, because getOriginContext should not be testing `isUK` or equivalent, anymore.